CHANGELOG.asciidoc

```diff
@@ -115,25 +137,17 @@ https://github.com/elastic/beats/compare/v6.2.3...v6.3.0[View commits]
 
 - De dot keys of labels and annotations in kubernetes meta processors to prevent collisions. {pull}6203[6203]
 - Rename `beat.cpu.*.time metrics` to `beat.cpu.*.time.ms`. {pull}6449[6449]
-- Mark `system.syslog.message` and `system.auth.message` as `text` instead of `keyword`. {pull}6589[6589]
-- Allow override of dynamic template `match_mapping_type` for fields with object_type. {pull}6691[6691]
 - Add `host.name` field to all events, to avoid mapping conflicts. This could be breaking Logstash configs if you rely on the `host` field being a string. {pull}7051[7051]
 
 *Filebeat*
 
 - Remove the undefined `username` option from the Redis input and clarify the documentation. {pull}6662[6662]
```

Comment:
This is also not a breaking change as the feature never existed I think.

Reply:
Moved to Added section.